tools/apiview/emitters/cadl-apiview/ci.yml

```diff
@@ -0,0 +1,105 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/apiview/emitters/cadl-apiview
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/apiview/emitters/cadl-apiview
+
+variables:
+  NodeVersion: '16.x'
+  CadlEmitterDirectory: 'tools/apiview/emitters/cadl-apiview'
+  ArtifactName: 'apiview'
+  FeedRegistry: 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-js/npm/registry/'
+
+stages:
+  - stage: 'Build'
+    jobs:
+      - job: 'Build'
+
+        pool:
+          name: azsdk-pool-mms-ubuntu-2004-general
+          vmImage: MMSUbuntu20.04
+
+        steps:
+          - task: NodeTool@0
+            inputs:
+              versionSpec: '$(NodeVersion)'
+            displayName: 'Use NodeJS $(NodeVersion)'
+
+          - script: |
+              npm install -g npm@8.16.0
```

Comment:
Should we be using the npm version that ships with the node version?

Reply:
yes we can. I can make that change and remove installation of 8.16

Reply:
Per @praveenkuttappan, he will address this in a follow-up PR where he will also switch the pipeline publishing from the dev feed directly to NPM.